src_assets/common/assets/web/config.html

```diff
@@ -105,6 +105,29 @@ <h1 class="my-4">Configuration</h1>
         </select>
         <div class="form-text">Choose which type of gamepad to emulate on the host</div>
       </div>
+      <div class="accordion" v-if="config.gamepad === 'ds4'">
```

Comment:
Does this auto update when the gamepad option is changed? I had an issue recently where I did something similar and the ui didn't update upon change.

Reply:
It does! I can confirm that switching to auto and x360 will collapse this accordion from being visible in realtime.

Reply:
Great, thanks for confirming!